eventActions/afkMessageCheckAction.js

```diff
@@ -41,16 +41,33 @@ class afkMessageCheckAction {
 			.setColor('#FFEC09');
 		const user = message.author;
 
+		// Returns a single integer rounded down for the difference in minutes between two Date.now() timestamps
+		function timedifference(timestamp1, timestamp2) {
```

Comment:
remove timestamp1+timestamp2 from function call since you're redefining it immediately after